diff -Naur munin-2.0.18.orig/master/lib/Munin/Master/GraphOld.pm munin-2.0.18/master/lib/Munin/Master/GraphOld.pm
--- munin-2.0.18.orig/master/lib/Munin/Master/GraphOld.pm	2014-03-17 22:23:34.000000000 +0400
+++ munin-2.0.18/master/lib/Munin/Master/GraphOld.pm	2014-03-17 22:23:40.000000000 +0400
@@ -1670,9 +1670,9 @@
 	# The RRD 1.2 documentation says you can identify font family
 	# names but I never got that to work, but full font path worked
 	@options = (
-		'--font', "LEGEND:7:$libdir/DejaVuSansMono.ttf",
-		'--font', "UNIT:7:$libdir/DejaVuSans.ttf",
-		'--font', "AXIS:7:$libdir/DejaVuSans.ttf",
+		'--font', "LEGEND:7:/usr/share/fonts/TTF/dejavu/DejaVuSansMono.ttf",
+		'--font', "UNIT:7:/usr/share/fonts/TTF/dejavu/DejaVuSans.ttf",
+		'--font', "AXIS:7:/usr/share/fonts/TTF/dejavu/DejaVuSans.ttf",
 	       );
     } else {
 	# At least 1.3
